alibi_detect/od/processor/parallel.py

```diff
@@ -0,0 +1,26 @@
+import ray
+import numpy as np
+
+from alibi_detect.od.processor.base import BaseProcessor
+
+
+class ParallelProcessor(BaseProcessor):
+    def __init__(self, detectors):
+        self.detectors = detectors
+    
+    def __call__(self, X):
+        """This might not be the best way to do this. Need to checK:
+        1. Does ray serlize each detector as we pass it to pfunc.
+        2. Is this an issue?
```

Comment:
Yea, iirc `ray` uses `pickle` to serialize arguments so it won't work for all detectors out--of-the-box (e.g. unpicklable `tensorflow` detectors).